todo-mvc/package.json

```diff
@@ -15,10 +16,24 @@
     "node": ">=6",
     "npm": ">=3"
   },
+  "peerDependencies": {
```

Comment:
@lzhoucs these should have stayed as dependencies. As they are actual dependencies of the TODO app